File: Day15/power_calculator.py
"""program to find power of a Number"""

def calculate_power(base,exponent):
    """Define the function to calculate the power of a number"""
    return base**exponent

base_value = float(input("Enter base: "))

exponent_value = float(input("Enter exponent: "))

res = calculate_power(base_value,exponent_value)

print(f"{base_value} raised to the power of {exponent_value} is {res}")

File: Day15/powerCalculator.js
function calculatePower(base,exponent){ 

    let res = 1.0;
    if(exponent >= 0){
        for(let i =1; i<= exponent; i++){
            res *= base;
        }
    }else{
        for(let i =1;i <= (-exponent);i++){
            res /= base;
       }
    }
    return res;
}
let base =parseInt(prompt("Enter the Base: "));

let exponent = parseInt(prompt("Enter the exponent: "));

console.log(`${exponent} raised to the power of ${base}: ${calculatePower(base,exponent)}`);
